mardi_importer/mardi_importer/zenodo/manualUpload.py

```diff
@@ -0,0 +1,13 @@
+from mardiclient import MardiClient
+from mardiclient import config
+
+mc = MardiClient(user="Rim", password="M!dight^2469")
```

Comment:
Not sure if this are valid credentials anywhere, but they should be deleted from here. Change your password also in case these are used somewhere.